cucumber-expressions/javascript/scripts/postinstall.js

```diff
@@ -0,0 +1,10 @@
+var path = require('path')
+var fs = require('fs')
+var exec = require('child_process').exec
+
+fs.exists(path.join(__dirname, '..', 'dist'), function (err) {
+  if (err) throw err;
+  exec('npm run build', {cwd: path.join(__dirname, '..')}, function (err) {
+    if (err) throw err;
+  })
+})
```

Comment:
I think this should be removed. The build script requires dev dependencies and this will run on production installs as well. Since you have the prepublish script setup, I don't understand why this would be need